src/auth/roles/index.ts

```diff
@@ -85,15 +85,18 @@ const Roles: RolesInterface = {
   },
 
   userHas: async (client, { role, userId, organisationId }) => {
+    const sub = Array.isArray(role)
+      ? client('access_role') .select('access_role_id') .whereIn('access_role_name', role)
```

Comment:
Why are you using `whereIn`, are you expecting `role` to be an array?

Reply:
Yes, hence `Array.isArray`.

Reply:
Can it be changed to `roles`?

Reply:
It's not always an array, hence the ternary.

Reply:
Thought it read better as plural, but feel free to leave.